src/components/tooltip/use-tooltip-positioning.js

```diff
@@ -12,7 +12,7 @@ export default function useTooltipPositioning({
   const hide = () => setTarget(null);
 
   useLayoutEffect(() => {
-    if (!target) {
+    if (!target || !tooltipRef.current) {
```

Comment:
Interesting. I wonder why the ref is not set up by now. 🤔 The effect is ran after painting to the DOM so all DOM refs should be present. 

Can you add a bit more information about how you found this bug?

Reply:
It does work correctly in the initial case, but (and this is how I found it), if you pass `disabled={true}` to the `Tooltip` after it is opened, or otherwise remove it from the DOM while it is being shown, this case shows up.